handbook/engineering/scaling-fleet.md

```diff
@@ -69,7 +69,7 @@ However, this database feature doesn’t come without a cost. The one to focus o
 
 The TLDR is: understand very well how a table will be used. If we do bulk inserts/updates, InnoDB might lock more than you anticipate and cause issues. This is not an argument to not do bulk inserts/updates, but to be very careful when you add a foreign key.
 
-In particular, host_id is a foreign key we’ve been skipping in all the new additional host data tables, which is not something that comes for free, as with that, [we have to keep the data consistent by hand with cleanups](https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/hosts.go#L309-L309).
```

Comment:
You should add a permalink instead:
![Screenshot 2023-10-04 at 16 21 58](https://github.com/fleetdm/fleet/assets/2073526/5f4591c5-682c-43d0-a0c5-4f663e4510c8)